resources/lang/en/Config.properties

```diff
@@ -1,5 +1,6 @@
 ConfigurationTab=Plugin Configuration
 isActiveLabel=Enable
+termsAndConditions=Accept Terms and Conditions listed here 
```

Comment:
@kraath
how can we place a link in this file that will lead to our terms and conditions page. ?